### PR TITLE
Add UrlFetch Example

### PR DESCRIPTION
**Pull request by waprin**

### CODE REVIEW

File: appengine/urlfetch/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## App Engine UrlFetch Docs Snippets
+
+This sample application demonstrates different ways to request a URL
+on App Engine
+
+
```
Comment (jonparrott) [inline appengine/urlfetch/README.md]: auto-doc-links fences here, please.

File: appengine/urlfetch/main.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    def get(self):
+        url = "http://www.google.com/"
+        try:
+            result = urlfetch.fetch(url)
+            if result.status_code == 200:
+                self.response.write(result.content)
+            else:
+                self.response.status_code = result.status_code
+        except urllib2.URLError, e:
```
Comment (jonparrott) [inline appengine/urlfetch/main.py]: Wrong exception?

File: appengine/urlfetch/main.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    form_fields = {
+        "first_name": "Albert",
+        "last_name": "Johnson",
+    }
+
+    def get(self):
+        try:
+            form_data = urllib.urlencode(UrlPostHandler.form_fields)
+            headers = {'Content-Type': 'application/x-www-form-urlencoded'}
+            result = urlfetch.fetch(url="http://localhost:8080/submit_form",
```
Comment (jonparrott) [inline appengine/urlfetch/main.py]: please newline before url.

(I know I'm picking nits)

File: appengine/urlfetch/main.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+"""
+
+import urllib
+import urllib2
+import webapp2
+
+from google.appengine.api import urlfetch
+
+
+def handle_error(e):
```
Comment (jonparrott) [inline appengine/urlfetch/main.py]: What's this here for?

File: appengine/urlfetch/main.py [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    def get(self):
+        url = "http://www.google.com/"
+        try:
+            result = urlfetch.fetch(url)
+            if result.status_code == 200:
+                self.response.write(result.content)
+            else:
+                self.response.status_code = result.status_code
+        except urllib2.URLError, e:
+            raise e
```
Comment (jonparrott) [inline appengine/urlfetch/main.py]: might as well log the error if you're going to catch it and not do anything with it.

File: appengine/urlfetch/main.py [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+                                    method=urlfetch.POST,
+                                    headers=headers)
+            self.response.write(result.content)
+        except urllib2.URLError, e:
+            raise e
+
+
+class SubmitHandler(webapp2.RequestHandler):
+    """ Handler that receives UrlPostHandler POST request
+    """
```
Comment (jonparrott) [inline appengine/urlfetch/main.py]: will this not fit on the line above?

File: appengine/urlfetch/README.md (hunk 1 above)
Comment (waprin) [inline appengine/urlfetch/README.md]: done

File: appengine/urlfetch/main.py (hunk 4 above)
Comment (waprin) [inline appengine/urlfetch/main.py]: mostly keeping consistent with existing sample but replaced with logging

File: appengine/urlfetch/main.py (hunk 2 above)
Comment (waprin) [inline appengine/urlfetch/main.py]: done

File: appengine/urlfetch/main.py (hunk 5 above)
Comment (waprin) [inline appengine/urlfetch/main.py]: done

File: appengine/urlfetch/main.py (hunk 3 above)
Comment (waprin) [inline appengine/urlfetch/main.py]: done

**Comment by jonparrott** [comment]
LGTM, pending travis.

**Pull request merged**